gobcore/datastore/sql.py

```diff
@@ -27,6 +27,10 @@ def write_rows(self, table: str, rows: List[list]) -> None:
     def execute(self, query: str) -> None:
         pass  # pragma: no cover
 
+    @abstractmethod
+    def copy_expert(self, query: str, data: str) -> None:
```

Comment:
I think copy_export is specific for postgresql?

Reply:
I think so! The copy_expert(query, data) used is imported from psycopg2 lib as the execute() function.

Reply:
yes, so the abstractmethod can be removed here ;) (oracle is for instance also a subclass)

Reply:
een goede! Done.